be/src/vec/exec/join/process_hash_table_probe_impl.h

```diff
@@ -839,7 +865,8 @@ struct ExtractType<T(U)> {
     template Status                                                                           \
     ProcessHashTableProbe<JoinOpType>::process_data_in_hashtable<ExtractType<void(T)>::Type>( \
             ExtractType<void(T)>::Type & hash_table_ctx, MutableBlock & mutable_block,        \
-            Block * output_block, bool* eos)
+            Block * output_block, bool* eos,                                                  \
+            ForwardIterator<RowRefListWithFlag>& mapped_visited_iter)
```

Comment:
do not need pass a arg ? better use _join_node to get the value

Reply:
fixed